### PR TITLE
Added initial loading of markers, pin for address, and directions link/textbox

### PR DESCRIPTION
**Pull request by nnarang7**
Added initial loading of markers, pin for address, and directions link/textbox

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -600,8 +604,8 @@ def remove_food_resource_type():
 	return jsonify(success="success")
 
 
-@app.route('/_search_query', methods=['GET', 'POST'])
-def save_search_query():
+@app.route('/_search_query_zipcode', methods=['GET', 'POST'])
+def save_search_query_zipcode():
```
Comment (weronica) [inline app/views.py]: Why did you rename this function and route?

File: app/templates/newmaps.html [hunk 2]
```diff
@@ -155,4 +252,4 @@ <h1 class="page-header" id="map-header">Philly Food Finder</h1>
 <!-- FAQ -->
 {% block content_faq %}
```
Comment (weronica) [inline app/templates/newmaps.html]: Can you remove the `content_faq` block and corresponding end block? We are no longer using it.

File: app/templates/newmaps.html [hunk 3]
```diff
@@ -77,10 +172,10 @@
                       var newMarker = results[0].geometry.location; 
                       myLat = newMarker.lat();
                       myLng = newMarker.lng();
-                      console.log(myLat);
-                      console.log(myLng);
-                      console.log(newTitle);
-                      console.log(image);
+                      var addressInfo = results[0].formatted_address;
+                      var removedCommas = addressInfo.split(",").join("");
+                      var linkInfo = removedCommas.split(" ").join("+");
```
Comment (weronica) [inline app/templates/newmaps.html]: Could you provide a comment explaining what you're doing in lines 175-177?

File: app/templates/newmaps.html [hunk 4]
```diff
@@ -8,62 +9,156 @@
...
         },
             function(data) {
+              
+              for (var j = data["addresses"].length; j >=0; j--)
+              {
+                var food_resource_id = data["addresses"][j]["food_resource_id"];
+                if (($("#map-filter-by-seniors").is(':checked')) && 
+                  data["addresses"][j]["food_resource_id"]          )
+              }
+              if ($("#map-filter-by-seniors").is(':checked')) //filtering results by checkbox categories
```
Comment (weronica) [inline app/templates/newmaps.html]: Is this checkbox code currently functional? If not, can you comment it out and explain what it's current behavior is.

File: app/templates/newmaps.html [hunk 5]
```diff
@@ -8,62 +9,156 @@
...
+        {
+          queryString += "_seniors";
+        }
+        if ($("#map-filter-by-seniors").is(':checked'))
+        {
+          queryString += "_seniors";
+        }
+        if ($("#map-filter-by-seniors").is(':checked'))
+        {
+          queryString += "_seniors";
```
Comment (weronica) [inline app/templates/newmaps.html]: Is this checkbox code currently functional? If not, can you comment it out and explain what it's current behavior is.

File: app/views.py (hunk 1 above)
Comment (nnarang7) [inline app/views.py]: I was going to have a different search query given a zipcode or address, but then i realized that there was a better way of doing that, and i forgot to change the search_query back to the previous form.

File: app/templates/newmaps.html (hunk 2 above)
Comment (nnarang7) [inline app/templates/newmaps.html]: Removed!

File: app/templates/newmaps.html (hunk 5 above)
Comment (nnarang7) [inline app/templates/newmaps.html]: I added the comments for the checkbox behavior and search queries

File: app/views.py (hunk 1 above)
Comment (weronica) [inline app/views.py]: Ok, sounds good.

File: app/templates/newmaps.html (hunk 5 above)
Comment (weronica) [inline app/templates/newmaps.html]: It looks like you still need to commit and push those changes to this branch. You'll also need to pull from master again.

Comment (nnarang7) [inline app/templates/newmaps.html]: I pushed the changes to the branch and pulled from master.
